backend/read_env.py

```diff
@@ -57,8 +57,8 @@ def get_argument_from_env() -> Tuple[str, list, list, dict, str]:
         if rocm_root:
             cmake_args.append(f"-DCMAKE_HIP_COMPILER_ROCM_ROOT:STRING={rocm_root}")
         hipcc_flags = os.environ.get("HIP_HIPCC_FLAGS")
```

Comment:
I find this flag is not documented, could you please add it to the doc? Thanks!

Reply:
I don't prefer using this environment variable, though. The CMake natively supported [`HIPFLAGS`](https://cmake.org/cmake/help/latest/envvar/HIPFLAGS.html) should be used instead of our customized flag. The old flag is kept for compatibility. I'll add `HIPFLAGS` instead.